nodes/babe-grandpa-node/Cargo.toml

```
@@ -0,0 +1,60 @@
+[package]
+name = "babe-grandpa-node"
+version = "2.0.0-alpha.6"
+edition = "2018"
+authors = ['Joshy Orndorff', '4meta5', 'Jimmy Chu']
+homepage = 'https://substrate.dev/recipes'
+repository = 'https://github.com/substrate-developer-hub/recipes'
+description = 'A Substrate node with Babe and Grandpa authorities taken from the runtime.'
```

```suggestion
description = 'A Substrate node with Babe and Grandpa authorities executed in the runtime.'
```
?

Not sure I understand what "taken" means here (might just be me though).

---

Rephrased to:

> A Substrate node where consensus authorities are determined by the runtime and passed to the consensus engine via a runtime API.

Good catch.